onnxruntime/hosting/http/util.cc

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <iostream>
+#include <boost/beast/core.hpp>
+#include "util.h"
+
+namespace onnxruntime {
+namespace hosting {
+
+// Report a failure
+void ErrorHandling(beast::error_code ec, char const* what) {
```

Comment:
Ehhhh does this really need to be broken up into a header and .cc?

Reply:
For now, no. But I would some more functions will be here when we implement controllers.